lua/image/backends/kitty/helpers.lua

```diff
@@ -92,6 +92,7 @@ local write_graphics = function(config, data)
       else
         control_payload = "m=1"
       end
+      vim.loop.sleep(1)
```

Comment:
`vim.loop` is being deprecated, you should use `vim.uv` instead.

Reply:
looks great otherwise @SigmaRichards